README.md

```diff
@@ -21,11 +27,13 @@ A simple command that runs the demo. For example,
 python <demo_name>.py
 ```
 
+If your example requires user input, make sure to specify any input limitations.
+
 ## Code Overview
 
 A general overview of how the code works.
 
-Prefer bite-sized descriptions in bullet points:
+We prefer bite-sized descriptions in bullet points:
```

Comment:
I think you mean "We prefer descriptions in bite-sized bullet points"
We're fine with lengthy, comprehensive description, just we want it broken into comprehensible chunks